### DISCUSSION TITLE
Individual layout view for different folders in file library

### DISCUSSION THREAD

**Discussion by neilcamilleri** [directus/directus#3463] [Feature Requests]
Some folders may be for pictures, some might be for documents. For one, you may want the card view and the other the table view. Currently, only one layout view is possible for folders in your file library.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
